### DISCUSSION TITLE
qin-cdc 实时同步mysql数据到doris

### DISCUSSION THREAD

**Discussion by liuxinwang** [apache/doris#36694] [Show and tell]
[qin-cdc github](https://github.com/sqlpub/qin-cdc) Real-time data synchronization component, supports MySQL -> StarRocks, MySQL, Doris(TODO Kafka, etc.), used to replace Aliyun DTS

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
